docs/features/nullable-reference-types.md

```diff
@@ -70,6 +70,13 @@ _Describe set of warnings. Differentiate W warnings._
 ### Null tests
 _Describe the set of tests that affect flow state._
 
+## `default`
+`default(T)` is `T?` if `T` is a reference type. _Should `default(T?)` be an error?_
```

Comment:
>reference type [](start = 33, length = 14)

It may be good to explain the meaning of `T?`, especially for unconstrained `T`.
Also, the phrasing may need to be adjusted "is a reference or unconstrained type".

Tangent: the null-state of `null` and the null-state of an instantiation (`new`) are trivial.

This spec change is unrelated to the PR. It may be better to have a separate PR for the spec change.

Reply:
Added a question about unconstrained type parameters.

---
In reply to: [195513298](https://github.com/dotnet/roslyn/pull/27525#discussion_r195513298) [](ancestors = 195513298)